decrypt_oracle/src/aws_encryption_sdk_decrypt_oracle/key_providers/counting.py

```diff
@@ -34,7 +34,7 @@ def __init__(self) -> None:
 
 
 class CountingMasterKey(MasterKey):
-    """Master key that generates deterministic data keys and decrypts a pre-defined
+    r"""Master key that generates deterministic data keys and decrypts a pre-defined
```

Comment:
I haven't seen this one before, a string prefixed with r. Is it a python thing or a typo?

Reply:
https://stackoverflow.com/questions/4780088/what-does-preceding-a-string-literal-with-r-mean